Makefile

```diff
@@ -198,7 +198,8 @@ ginkgo-tests:
 	    --covermode atomic \
 	    --output-dir $(COVERAGE_PATH) \
 	    --junit-report junit.xml \
-	    --coverprofile coverprofile \
+	    --coverprofile $(COVERAGE_PATH)/coverprofile \
```

Comment:
nitpick/no need to update this now, just a comment for future reference: This change is unrelated to fixing the  missing sysfs testdata. Therefore IMO, it should be a separate commit of its own, ideally committed with something like `git commit -s -m "Makefile: fix ginkgo coverprofile generation."` or something to that effect. Then if those two commits then were files in a single PR (I can see the justification for that), a more suitable title could be something like "test: fix ginkgo unit test rule, missing test data."

Again, no need to change it now. Just trying to describe how we usually prefer to go with commits (each commit a single self-contained logical unit of change) and PRs.

Reply:
I should admit I'm really bad with separating the commits but I will follow your instructions next time.